### DISCUSSION TITLE
Файл service_install.bat вылетает сразу при запуске

### DISCUSSION THREAD

**Discussion by oski4hh** [Flowseal/zapret-discord-youtube#520] [General]
На доли секунды появляется окошко консоли и сразу же закрывается

**Comment by WhatTheShawod** [comment]
Аналогичная проблема.
Только у меня даже консоль не мелькает, а только её иконка в таскбаре.

**Comment by FatCatBueno** [comment]
+

**Comment by Flowseal** [comment]
Создавайте issue с конкретными ошибками (ошибки можно увидеть, открыв с помощью cmd от имени администратора, самостоятельно прописав путь до файла)
